### PR TITLE
Remove backend cart

### PR DESCRIPTION
**Pull request by mbj**
Removes:

* The ability to create new orders from the backend
* Editing the order in "cart" view (line items).

Eliminates:

* A very indeterministic backend spec.

### CODE REVIEW

**Comment by mbj** [comment]
@dkubb This is ready for review.

File: backend/app/controllers/spree/admin/orders_controller.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@ module Spree
   module Admin
     class OrdersController < Spree::Admin::BaseController
       before_action :initialize_order_events
-      before_action :load_order, only: [:edit, :update, :cancel, :resume, :approve, :resend, :open_adjustments, :close_adjustments, :cart]
+      before_action :load_order, only: [:edit, :update, :cancel, :resume, :approve, :resend, :open_adjustments, :close_adjustments]
```
Comment (dkubb) [inline backend/app/controllers/spree/admin/orders_controller.rb]: This should probably use `%i[...]`.

**Comment by dkubb** [comment]
@mbj I flagged one small issue, but otherwise this change is fine. Are we sure that none of this is being used?

**Comment by mbj** [comment]
@dkubb Ready for the next pass.

**Comment by dkubb** [comment]
@mbj I am fine with this change. Will merge on green.

**Pull request merged**